membrane_curvature/tests/test_membrane_curvature.py

```diff
@@ -401,23 +481,23 @@ def test_analysis_get_z_surface_wrap_xy(self, universe_dummy_wrap_xy, dummy_surf
     # test mean curvature
     def test_analysis_mean_wrap(self, curvature_unwrapped_universe, dummy_surface):
         avg_mean = curvature_unwrapped_universe.results.average_mean
-        expected_mean = mean_curvature(dummy_surface)
+        expected_mean = mean_curvature(dummy_surface, curvature_unwrapped_universe.dy, curvature_unwrapped_universe.dx)
```

Comment:
Parametrize over these arguments, to check that the original implementation still functions as intended. 

```
@pytest.mark.parametrize("varargs", [, [], [dy, dx]])
``
Or if the parametrization is too complicated split into two tests.

Reply:
Not sure what to do here. The original implementation is fundamentally flawed. 

The test fixture `universe_dummy_wrap` has a specific grid spacing and system dimension. Taking derivatives with respect to the default `dx`, `dy` value (=1) is like saying:

$f'(x) = \frac{f(x+h) -f(x)}{1}$.

When the derivatives must be taken with respect to the spacing, $h$:

$f'(x) = \frac{f(x+h) -f(x)}{h}$.

To have `dx`, `dy` as parameters, the fixture would have to be changed to produce meshes with varied discretization.